spec/integration/outputs/ilm_spec.rb

```diff
@@ -198,12 +198,16 @@
     let (:settings) { super().merge!({ 'ilm_enabled' => true }) }
 
     it 'should raise a configuration error' do
+      # TODO should be refactored not to rely on plugin internals
```

Comment:
integration test was relying on a raise from the other thread to propagate, which is problematic.
the new logic to rescue everything from the async thread and propagate exceptions back as `thread.value` - making it a bit hard to test -> spec example should be rewritten or migrated to a unit test.